### PR TITLE
Bumps in garden : ign-gui7

### PR DESCRIPTION
**Pull request by methylDragon**
See https://github.com/ignition-tooling/release-tools/issues/685

### CODE REVIEW

File: src/plugins/world_control/WorldControl.cc [hunk 1]
```diff
@@ -64,7 +64,7 @@ namespace plugins
 
     /// \brief Whether server communication should occur through an event (true)
     /// or service (false). The service option is used by default for
-    /// ign-gui6, and should be changed to use the event by default in ign-gui7.
+    /// ign-gui7, and should be changed to use the event by default in ign-gui7.
```
Comment (chapulina) [inline src/plugins/world_control/WorldControl.cc:67]: Oops

**Review by chapulina — APPROVED** [review]
Just a minor greedy match. LGTM otherwise, once it has 🟢 CI

File: src/plugins/world_control/WorldControl.cc (hunk 1 above)
Comment (methylDragon) [reply src/plugins/world_control/WorldControl.cc]: https://github.com/ignitionrobotics/ign-gui/pull/385/commits/a007fef08d52bec8a701f1737e023d03d9009cf5

**Comment by chapulina** [comment]
The new test failures are caused by a bad combination of `ign-cmake2` and `ign-cmake3` and their `utilities` package. I'm going to merge this as is because:

* We have various merged and open PRs to get rid of `utilities`
* The next wave of bumps should remove the double `ign-cmake2` and `ign-cmake3`

Let's not block on this, merging!

**Pull request merged**